src/VideoStreaming/VideoReceiver.cc

```diff
@@ -923,6 +939,30 @@ VideoReceiver::_unlinkCallBack(GstPad* pad, GstPadProbeInfo* info, gpointer user
 }
 #endif
 
+//-----------------------------------------------------------------------------
+#if defined(QGC_GST_STREAMING)
+GstPadProbeReturn
+VideoReceiver::_videoSinkProbe(GstPad* pad, GstPadProbeInfo* info, gpointer user_data)
+{
+    Q_UNUSED(pad);
+    if(info != nullptr && user_data != nullptr) {
+        VideoReceiver* pThis = static_cast<VideoReceiver*>(user_data);
+        pThis->_noteVideoSinkFrame();
+    }
+
+    return GST_PAD_PROBE_OK;
+}
+#endif
+
+//-----------------------------------------------------------------------------
+#if defined(QGC_GST_STREAMING)
+void
+VideoReceiver::_noteVideoSinkFrame()
+{
+    _lastFrameTime = time(nullptr);
```

Comment:
QTime::currentTime().msecs() ? (the readability of time(NULL), time(0) and time(nullptr) are really bad.

Reply:
OK - I'll switch to QTime as it seems whole project is using that.

Reply:
QTime is not suitable for that. Used QDateTime::currentSecsSinceEpoch() to archive what is need. In fact time() and QDateTime::currentSecsSinceEpoch() call the same gettimeofday() on POSIX system (see https://code.woboq.org/userspace/glibc/sysdeps/posix/time.c.html#31 and https://github.com/qt/qtbase/blob/dev/src/corelib/time/qdatetime.cpp#L4993). The difference is just in amount of characters that one needs to write to call one of them (and to read that code - 13 vs 34 chars).

Reply:
thanks. :) consistency usually is better than number of lines.